remote-state.tf

```diff
@@ -0,0 +1,20 @@
+resource "random_string" "resource_code" {
+  length  = 5
+  special = false
+  upper   = false
+}
+
+resource "azurerm_storage_account" "tfstate" {
+  name                     = "tfstate${random_string.resource_code.result}"
```

Comment:
You don't normally manage Terraform's backend state with Terraform. It means you can't recreate everything with one terraform apply, and can't use workspaces for different environments.

If you delete this file you'll need to remove these objects from the state so Terraform doesn't destroy them, e.g. `terraform state rm 'azurerm_storage_account.tfstate'`.